lib/drivers/tandemDriver.js

```diff
@@ -1627,6 +1627,29 @@ module.exports = function (config) {
     return records;
   };
 
+  var buildSuspendResumeRecords = function (data, records) {
+    var suspendResumeRecords = filterLogEntries([PUMP_LOG_RECORDS.LID_PUMPING_SUSPENDED,PUMP_LOG_RECORDS.LID_PUMPING_RESUMED], data.log_records);
+
+    suspendResumeRecords.forEach(function (entry) {
+      var postrecord;
+
+      if (entry.header_id === PUMP_LOG_RECORDS.LID_PUMPING_SUSPENDED.value) {
+        postrecord = cfg.builder.makeDeviceEventSuspend()
+        .with_reason({suspended: 'manual'}); //TODO: look if we can infer reason from alarm, e.g. with occlusion
+      } else if (entry.header_id === PUMP_LOG_RECORDS.LID_PUMPING_RESUMED.value) {
+        postrecord = cfg.builder.makeDeviceEventResume()
+        .with_reason({resumed: 'manual'});
+      }
+
+      postrecord.with_deviceTime(entry.deviceTime);
+      postrecord.set('index', entry.log_index);
+      cfg.tzoUtil.fillInUTCInfo(postrecord, entry.jsDate);
```

Comment:
Same thing here as in #199 re: `uncertain-timestamp` because of bad bootstrapping. I'm letting you problem-solve/hunt it down :P